test/includes.js

```diff
@@ -29,6 +29,11 @@ describe('Includes', function () {
     done()
   })
 
+  it('should <include> a template but not parse variables(includes/includeNoParsing.html)', function (done) {
```

Comment:
this should go further than just not parsing the variables inside the template - it should not parse *any* teddy features, including `<if>` tags, `<loop>` tags, etc. 

we should add a test that has an include tag with a `noparse` attribute which includes a more complex templates that leverages more teddy features to test that it suppresses all teddy features and not just variables